tools/depends/target/Makefile

```diff
@@ -95,6 +95,10 @@ ifeq ($(NEED_LIBICONV),1)
   ICONV = libiconv
 endif
 
+ifeq ($(TARGET_WEBOS),1)
```

Comment:
There doesn't seem to be anyway to set this. Can `webos-wayland-extensions` just always be built with wayland/linux?

Reply:
we'd need to add it to depends configure